### PR TITLE
Add new fee columns to history transactions

### PR DESCRIPTION
**Pull request by chowbao**

### CODE REVIEW

File: schemas/history_transactions_schema.json [hunk 1]
```diff
@@ -178,5 +178,20 @@
     "mode": "NULLABLE",
     "name": "transaction_result_code",
     "type": "STRING"
+  },
+  {
+    "mode": "NULLABLE",
+    "name": "inclusion_fee_bid",
+    "type": "STRING"
```
Comment (sydneynotthecity) [inline schemas/history_transactions_schema.json:185]: should these be int?

**Review by sydneynotthecity — COMMENTED** [review]
I think there is a copy/pasta error in the new column definitions. Lmk if they are intentionally defined as string

File: schemas/history_transactions_schema.json (hunk 1 above)
Comment (chowbao) [reply schemas/history_transactions_schema.json:185]: Oh oops you're right

**Review by sydneynotthecity — APPROVED** [review]
LGTM

**Pull request merged**